glusterd2/commands/volumes/volume-options-get.go

```diff
@@ -0,0 +1,99 @@
+package volumecommands
+
+import (
+	"net/http"
+
+	restutils "github.com/gluster/glusterd2/glusterd2/servers/rest/utils"
+	"github.com/gluster/glusterd2/glusterd2/volume"
+	"github.com/gluster/glusterd2/glusterd2/xlator"
+	"github.com/gluster/glusterd2/glusterd2/xlator/options"
+	"github.com/gluster/glusterd2/pkg/api"
+
+	"github.com/gorilla/mux"
+)
+
+func volumeOptionsGetHandler(w http.ResponseWriter, r *http.Request) {
+
+	ctx := r.Context()
+
+	volname := mux.Vars(r)["volname"]
+
+	volinfo, err := volume.GetVolume(volname)
+	if err != nil {
+		status, err := restutils.ErrToStatusCode(err)
+		restutils.SendHTTPError(ctx, w, status, err)
+		return
+	}
+
+	optname, found := mux.Vars(r)["optname"]
+
+	if found {
+		opt, err := xlator.FindOption(optname)
```

Comment:
`xlator.FindOption` can return two kind of errors: `InvalidKeyError` and `OptionNotFoundError`. A `404` must be returned only for `OptionNotFoundError` and `400` for `InvalidKeyError`

Reply:
done